.github/workflows/build.yml

```diff
@@ -10,6 +10,17 @@ on:
     branches: [ master ]
 
 jobs:
+  verify-python:
+    name: Python code verification (src)
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        run: |
+          pip install black
```

Comment:
I see that black is released as binaries (https://github.com/psf/black/releases/tag/21.7b0), I'd rather use that than pip to install it if possible

Reply:
It is simpler in the pipeline to use `pip`. After the changes, this command was moved to the `makefile` as `make set-dev`